src/app/components/shared/event-map/event-map.component.html

```diff
@@ -0,0 +1,36 @@
+<baw-map
+  [markers]="markers()"
+  [markerOptions]="{ gmpClickable: true }"
+  (markerClicked)="handleMarkerClicked($event)"
+>
+  <ng-template #markerTemplate let-marker="marker">
+    <svg
+      class="marker-template"
+      xmlns="http://www.w3.org/2000/svg"
+      width="40"
+      height="40"
+      viewBox="0 0 40 40"
+    >
+      <circle
+        cx="20"
+        cy="20"
+        r="20"
+        fill="var(--baw-highlight-lighter)"
+        opacity="0.6"
+      />
+      <circle cx="20" cy="20" r="16" fill="var(--baw-highlight)" opacity="0.8" />
+      <text
+        class="marker-count"
+        x="20"
+        y="25"
+        text-anchor="middle"
+        font-size="12"
+        fill="var(--baw-highlight-contrast)"
+        letter-spacing="0.75px"
+        font-weight="bold"
```

Comment:
though i'd suggest the style based attributes should be omitted and replace with css styles